protoc-gen-kotlin/protoc-gen-kotlin-common/src/main/kotlin/pbandk/gen/FileBuilder.kt

```diff
@@ -20,6 +20,7 @@ open class FileBuilder(val namer: Namer = Namer.Standard, val supportMaps: Boole
             ?: ctx.fileDesc.options?.uninterpretedOption?.find {
                 it.name.singleOrNull()?.namePart == "kotlin_package"
             }?.stringValue?.array?.let(Util::utf8ToString)
+            ?: ctx.packageMappings[ctx.fileDesc.`package`]
```

Comment:
It might be useful to support mapping based on prefix matching rather than only equality matching (e.g. any package name that starts with `from.package1` gets remapped to `to.package1.*`). But I suppose we can add that later if we find a need for it.

Reply:
Yeah, let's add a wildcard when we need it.  I think it might be unexpected behavior the way it is currently implemented, tbh though.  If I say "com.google" should be "pbandk.google", I probably expect "com.google.api" to become "pbandk.google.api"... but I don't want to take that on at this time, as the work can be done later, and some other use case might not want that, so let's just wait and see.